src/core/modules/NebulaMgr.cpp

```diff
@@ -411,14 +411,50 @@ void NebulaMgr::init()
 
 	setTypeFilters(typeFilters);
 
-	// TODO: mechanism to specify which sets get loaded at start time.
-	// candidate methods:
-	// 1. config file option (list of sets to load at startup)
-	// 2. load all
-	// 3. flag in nebula_textures.fab (yuk)
-	// 4. info.ini file in each set containing a "load at startup" item
-	// For now (0.9.0), just load the default set
-	loadNebulaSet("default");
+	// specify which sets get loaded at start time.
+	const QString nebulaSetName=conf->value("astro/nebula_set", "default").toString();
+
+	// Load the list of unwanted references.
+	const QString allSetsUnwantedRefsRaw = conf->value("astro/nebula_exclude_references").toString(); // Later: read this from config.ini. Syntax: "setName:unrefA,unrefB,..."
+	QString allUnwantedReferencesFromOneSetRaw;
+	if (allSetsUnwantedRefsRaw.contains(';'))
+	{
+		// full specification: SetA:refA,refB;SetB:refC
+#if (QT_VERSION>=QT_VERSION_CHECK(5,14,0))
+		QStringList allSetsUnwantedRefs=allSetsUnwantedRefsRaw.split(';', Qt::SkipEmptyParts);
```

Comment:
You could define `SkipEmptyParts` as it was done in `StelSkyCultureMgr.cpp:43` and thus keep the old-Qt version semantically the same.

Reply:
I couldn't find the old symbol, thanks.